tests/foreman/cli/test_activationkey.py

```diff
@@ -1311,7 +1294,7 @@ def test_update_autoattach_2(self):
                     u'Activation key updated', result[0]['message'])
 
     @tier1
-    def test_negative_update_auto_attach(self):
+    def test_negative_update_autoattach(self):
```

Comment:
`negative` and `invalid` denote the same thing.  So I like to keep my original proposed name.